presto-hive/src/main/java/io/prestosql/plugin/hive/s3/HiveS3Config.java

```diff
@@ -59,6 +62,7 @@
     private String s3UserAgentPrefix = "";
     private PrestoS3AclType s3AclType = PrestoS3AclType.PRIVATE;
     private boolean skipGlacierObjects;
+    private String s3SignerClass;
```

Comment:
You also need to add tests to `io.prestosql.plugin.hive.s3.TestHiveS3Config`

Reply:
Done